server/src/main/java/org/elasticsearch/action/admin/indices/create/TransportCreatePartitionsAction.java

```diff
@@ -219,7 +220,15 @@ private ClusterState executeCreateIndices(ClusterState currentState, CreateParti
             // Use only first index to validate that index can be created.
             // All indices share same template/settings so no need to repeat validation for each index.
             Settings commonIndexSettings = createCommonIndexSettings(currentState, template);
-            shardLimitValidator.validateShardLimit(commonIndexSettings, currentState);
+            final int numberOfShards = INDEX_NUMBER_OF_SHARDS_SETTING.get(commonIndexSettings);
+            final int numberOfReplicas = IndexMetadata.INDEX_NUMBER_OF_REPLICAS_SETTING.get(commonIndexSettings);
+            final int numShardsToCreate = numberOfShards * (1 + numberOfReplicas) * indicesToCreate.size();
+            shardLimitValidator.checkShardLimit(numShardsToCreate, currentState)
+                .ifPresent(err -> {
+                    final ValidationException e = new ValidationException();
+                    e.addValidationError(err);
+                    throw e;
+                });
```

Comment:
That's a regression introduced in 5.3 with https://github.com/crate/crate/pull/13843/. Thanks for catching!